libtransact/src/state/merkle/sql/mod.rs

```diff
@@ -15,7 +15,499 @@
  * -----------------------------------------------------------------------------
  */
 
-mod migration;
+//! SQL-backed merkle-radix state implementation.
+
+pub mod backend;
+pub mod migration;
```

Comment:
Is this pub change intentional?

Reply:
Yes, in both cases, but I've moved the pub changes to the relevant commits